extensions/wikibase/src/org/openrefine/wikibase/qa/scrutinizers/FileNameScrutinizer.java

```diff
@@ -49,13 +48,12 @@ public void batchIsBeginning() {
 
     /**
      * Attempt of a local implementation of the file name normalization that is done in MediaWiki. Assumes a non-empty
-     * file name as input.
+     * file name as input. This assumes that some special characters have been replaced already: `:`, `/` and `\`
      * 
      * @return
      */
     protected String normalizeFileName(String filename) {
-        String replaced = filename.replaceAll(normalizedFileNameChars.pattern(), "-")
-                .replaceAll("_", " ");
+        String replaced = filename.replaceAll("_", " ");
```

Comment:
Since this method is only normalizing spaces and other normalization is done outside, it could be renamed to something like normalizeFileNameSpaces.